docs/security-monitoring.md

```diff
@@ -23,11 +23,14 @@ Organizations using the agency add-on have the option to disable security monito
 
 ### Configuring Notifications
 Every user receives security notifications by email for all packages they have access to by default.
-Users can unsubscribe either from individual packages, or from all security notifications, if they not wish to receive any email notifications.
+Users can unsubscribe either from individual packages or from all security notifications if they do not wish to receive any email notifications.
 
-In addition to registered users receiving security notifications by email, you can configure notifications to be sent
-to other email addresses, e.g. a security mailinglist, or to a Slack channel.
-Create a notification channel for the desired destination and configure which notifications it should receive in the organization's security settings.
+The following notification channels are supported:
+* Email: send notifications to a configured email addresses list
+* Slack webhook: receive notifications in your configured Slack channel
```

Comment:
This should really be "Slack Webhook" if we consider this a name the whole thing should be upper case, this may need to be fixed in our UI as well.

Reply:
It's "Slack Webhook" in the UI, updated it here.